btf/format.go

```diff
@@ -174,10 +174,18 @@ func (gf *GoFormatter) writeIntLit(i *Int) {
 	}
 
 	bits := i.Size * 8
-	if i.Encoding.IsSigned() {
-		fmt.Fprintf(&gf.w, "int%d", bits)
+	if i.Size > 8 {
+		if i.Encoding.IsSigned() {
+			fmt.Fprintf(&gf.w, "[%d]byte /* int%d */", i.Size, bits)
```

Comment:
The latest code seems to have changed quite a bit so it was changed to:
```golang
...
	case Signed:
		if i.Size > 8 {
			fmt.Fprintf(&gf.w, "[%d]byte /* int%d */", i.Size, i.Size*8)
		} else {
			fmt.Fprintf(&gf.w, "int%d", bits)
		}
	case Char:
		if i.Size != 1 {
			return fmt.Errorf("char with size %d", i.Size)
		}
		// BTF doesn't have a way to specify the signedness of a char. Assume
		// we are dealing with unsigned, since this works nicely with []byte
		// in Go code.
		fallthrough
	case Unsigned:
		if i.Size > 8 {
			fmt.Fprintf(&gf.w, "[%d]byte /* uint%d */", i.Size, i.Size*8)
		} else {
			fmt.Fprintf(&gf.w, "uint%d", bits)
		}
...
```

Reply:
The latest code seems to have changed quite a bit so it was changed to:
[format#L185](https://github.com/Benjamin-Yim/ebpf/blob/fix_uint128/btf/format.go#L185)

```golang
	case Unsigned, Signed:
		stem := "uint"
		if i.Encoding == Signed {
			stem = "int"
		}
		if i.Size > 8 {
			fmt.Fprintf(&gf.w, "[%d]byte /* %s%d */", i.Size, stem, i.Size*8)
		} else {
			fmt.Fprintf(&gf.w, "%s%d", stem, bits)
		}
```
Is this okay?

> No need to split these two cases, how about the following:
> 
> ```go
> stem := "uint"
> if i.Encoding.IsSigned() {
>   stem = "int"
> }
> 
> if i.Size > 8 {
>   ...
> } else {
>    fmt.Fprintf(&gf.w, "%s%d", stem, bits)
> }
> ```